### PR TITLE
add jscolor v2.0.4 with autoupdate config

### LINKED ISSUE #6172: [Request] Add jscolor
**Library name:** jscolor
**Git repository url:** https://github.com/EastDesire/jscolor
**npm package url(optional):** 
**License(s):** Dual licensing: Commercial or GNU GPL v3
**Official homepage:** http://jscolor.com
**Wanna say something? Leave message here:** Thank you (-:

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by dannyxx001**
@PeterDaveHello please help me check this pull request, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/EastDesire/jscolor
 * Official website (optional, not the repository): http://jscolor.com/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 45
   - Count of watchers: 3
   - Count of forks: 27
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #6172, cc @EastDesire

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@dannyxx001 Pleasr ask other intern2 members first.

**Comment by dannyxx001** [comment]
@pvnr0082t @noahcooper please help me check this pull request, thanks.

**Comment by PeterDaveHello** [comment]
@dannyxx001 Only one reviewer

File: ajax/libs/jscolor/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  "keywords": [
+    "jscolor",
+    "web color"
+  ],
+  "author": {
+    "name": "EastDesire",
+    "url": "https://github.com/EastDesire"
+  },
+  "license": "GPL",
```
Comment (PeterDaveHello) [inline ajax/libs/jscolor/package.json]: Please confirm that is GPL-1.0, 2.0 or 3.0

Comment (dannyxx001) [inline ajax/libs/jscolor/package.json]: OK, this is GPL-3.0, and I have updated it.

**Comment by PeterDaveHello** [comment]
Don't forget to rebase with our latest master branch next time.

**Pull request merged**